overview.qmd

```diff
@@ -236,21 +237,21 @@ image source: https://worace.works/2022/02/23/kicking-the-tires-flatgeobuf/
 
 ::: {.column width="50%"}
 
-* Vector data is traditionally stored as rows representing points, lines, or polygons with an attribute table
+* Vector data is traditionally stored as rows representing points, lines, or polygons with an attribute table.
 * GeoParquet defines how to store vector data in Apache Parquet, which is a columnar storage format (like many cloud data warehouses). “Give me all points with height greater than 10m”.
-* Highly compressed
-* Single-file or multi-file
-* Recent support added to geopandas as a distinct function, R support with geoarrow
-* Potential for cross language in-memory shared access
-* Specifications for spatial-indexing, projection handling, etc. are still in discussion
+* Highly compressed.
+* Single-file or multi-file.
+* Recent support added to GeoPandas as a distinct function, R support with GeoArrow.
```

Comment:
```suggestion
* Geoparquet support was recently added to GeoPandas and is supported in R with GeoArrow.
```

Who can verify the R support? I'm not entirely sure what it means it is supported with GeoArrow as that is the in-memory format. Can Geoparquet be loaded in R?